src/utils/validate-connection.ts

```diff
@@ -7,10 +7,10 @@ import * as dotenv from 'dotenv';
 
 const main = async () => {
   dotenv.config();
-
+  const server =
+    process.env.API_ENDPOINT_PRIVATE_GRAPHQL || DEFAULT_ENDPOINTS.alkemioServer;
```

Comment:
I would not change the env var name as it is likely being passed in all the environments. So leave as ALKEMIO_SERVER_ENDPOINT as that is what is used in the environments....